recipes/cgns/all/test_package/CMakeLists.txt

```diff
@@ -1,8 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
 project(test_package CXX)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+find_package(CGNS REQUIRED)
 
 add_executable(test_package test_package.cpp)
-target_link_libraries(test_package ${CONAN_LIBS})
+target_link_libraries(test_package CGNS::CGNS)
```

Comment:
Because there are components I assume the headers are now being passed to the global target 🤔

Reply:
I don't follow.
Is CGNS::CGNS the "global target" ?
What should be changed?

Reply:
@prince-chrismc when you have time please, I'm not sure what you mean

Reply:
Sorry for not being more clear, I was trying to see why the build fail when I left that comment...

CI failed because the header was not found, but I can see it being placed in the package... I am not sure why that is I was just tossing a suggestion... might be a bug that was fix in 1.50 thats not in CCI

🤷 sorry I am not very helpful 🤗 💟

Reply:
Ok thanks mate.
@SSE4 could you suggest someone who could help with this?  (I'm not sure who I should tag as Triage Person)